apps/datafeeder/src/app/presentation/components/data-import-validation-map-panel/data-import-validation-map-panel.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="flex flex-col h-full">
+  <div class="flex flex-col justify-center h-16">
+    <span translate class="header-label font-bold pl-6 text-xl">
+      {{ headerLabel }}
+    </span>
+  </div>
+  <div class="flex flex-1 relative" #map>
+    <div
```

Comment:
don't go for hfull if not necessary.
Also add a padding, and maybe use `small` class if it exists, the text is too big.

The properties overlay mask a bit the feature, but I think it's not easy to position it correctly.